Rewamp the README and the documentation (#89)
jolicode/castor: doc/01-installation.md

# Installation and Autocomplete

## Installation

> **Note**
> Please note that castor needs PHP >= 8.1 to run.

### As a phar - recommended way

You can download the latest release of Castor as a phar file from the [releases
page](https://github.com/jolicode/castor/releases).

You can also download the latest version by browsing [the build
page](https://github.com/jolicode/castor/actions/workflows/build-phar.yml) and
selecting the last build.

We provide different phar for Linux / MacOS / Windows architectures to offer lighter phar
files. Download the correct one and make it available in your shell:

Example for Linux:
```bash
mv castor.linux-amd64.phar $HOME/.local/bin/castor
```

> ### Globally with Composer

Reviewer: ```suggestion

### Globally with Composer
```

# Installation and Autocomplete

## Installation

> **Note**
> Please note that castor needs PHP >= 8.1 to run.

### As a phar - recommended way

You can download the latest release of Castor as a phar file from the [releases
page](https://github.com/jolicode/castor/releases).

You can also download the latest version by browsing [the build
page](https://github.com/jolicode/castor/actions/workflows/build-phar.yml) and
selecting the last build.

> We provide different phar for Unix/Windows architectures to offer lighter phar

Reviewer: ```suggestion
We provide different phar for Linux / MacOS / Windows architectures to offer lighter phar
```

files. Download the correct one and make it available in your shell:

Example for Linux:
```bash
mv castor.linux-amd64.phar $HOME/.local/bin/castor
```

> 
> There are other ways to install Castor, please refer to the documentation
> 

Reviewer: ```suggestion

```
